### ISSUE TITLE
'active_support' dependency missing

### ISSUE THREAD

**Issue by jniederm** [oVirt/ovirt-site#423]
### Description of the problem
`./run-server.sh` executes `fetch-dashboard.rb` that requires `active_support`. However this gem is never installed. This causes following warning during each server start:

```
/usr/share/rubygems/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- active_support (LoadError)
	from /usr/share/rubygems/rubygems/core_ext/kernel_require.rb:55:in `require'
	from fetch-dashboard.rb:12:in `<main>'
```

**Comment by mureinik** [comment]
@duck-rh can't this issue be closed now?

**Comment by duck-rh** [comment]
I probably forgot the close message or did not format it well, but this was solved by PR #424

**Comment by mureinik** [comment]
Thanks @duck-rh
